cli/ops/fs.rs

```diff
@@ -248,14 +248,21 @@ async fn op_seek_async(
   _zero_copy: BufVec,
 ) -> Result<Value, ErrBox> {
   let (rid, seek_from) = seek_helper(args)?;
-  // TODO(ry) This is a fake async op. We need to use poll_fn,
```

Comment:
There is a trait for this as described.
Is it critical to use this (future::Poll) over just calling `seek.await` @ry ?